### DISCUSSION TITLE
Input Group

### DISCUSSION THREAD

**Discussion by AlexHedley** [microsoft/fluentui-blazor#1665] [Ideas]
Any plans on implementing an Input Group?

Unless I'm missing a way currently?

Bootstrap example

![image](https://github.com/microsoft/fluentui-blazor/assets/1573469/c9472b46-1fc0-4542-9490-35642e955c75)

https://getbootstrap.com/docs/5.3/forms/input-group/

**Comment by vnbaaij** [comment]
FluentTextField has 'start' and 'end' slots that you can use to, for example, place an icon in (see  https://fluentui-blazor.net/TextField#icons). By using the 'slot' in other tags, you could potentially also place other content in those spots.
I don't think we will go further than that as this concept does not exist in the Fluent Design System.

**Comment by AlexHedley** [comment]
Thanks. I’ll give that a go :-).
